source/learners/sfd_discussions/find_follow_conversations.rst

```diff
@@ -0,0 +1,155 @@
+
+3. **Topics:** Displays all discussion topics in the course. Topics follow the same three-level hierarchy as the course content:
+   
+   a. The top level represents course sections.  
+   b. The second level shows subsections.  
+   c. The third level lists units, which are the actual discussion topics that contain posts. Select a subsection to expand it and see its topics. When you click a topic name, you will see the posts related to it.
+
+4. **Learners:** Shows everyone who has written a post, response, or comment. Click a username to see their contributions. Under each name you can see the number of posts they created and their total contributions.
+
+.. tip:: Use *My Posts* for tracking your own activity and *Topics* when you want to focus on the part of the course you are studying.
+
+
+.. figure:: /_images/learners/Discussions_page.png
+   :alt: Screenshot of the discussion page
+   :width: 600px
+   :align: center
+
+   The Discussion page with navigation pane, content list, and content view areas.
+
+
+
+.. figure:: /_images/learners/Discussions_topics.png
+   :alt: How to navigate topic hierarchy.
+   :width: 600
+   :align: center
+
+   Navigating the 3-level topic hierarchy to view posts beloning to a topic.
+
+.. _Discussions inside course units:
+
+*********************************
+Discussions inside course units
+*********************************
+
+Some course units include a discussion panel on the right side of the page. It shows only the posts related to that
+unit’s topic, giving you a focused view of the conversation. Any post you create here also appears on the main
+Discussions page.
+
+.. tip:: Use the inline panel when you want to talk about something shown on the page.
+   Use the main Discussions tab when you want to search or browse across the entire course.
+
+
+.. figure:: /_images/learners/Discussions_sidebar.png
+   :alt: Discussions sidebar can be seen on right side of course content when enabled.
+   :width: 600
+   :align: center
+
+   Discussions sidebar can be seen on right side of course content when enabled.
+
+
+.. _Searching and filtering:
+
+************************
+Searching and filtering
+************************
+
+The search field at the top of *Discussions* page lets you search for:
+
+1. Discussion content when you have *All Posts* or *My Posts* tab open
+2. Discussion topics when you have *Topics* tab open
+3. Discussion users when you have *Learners* tab open. You can search by full or partial usernames.
+
+
+.. figure:: /_images/learners/Discussions_search.png
+   :alt: Search bar on top of *Discussions* page.
+   :width: 400
+   :align: center
+
+   Use the search bar to search for content, topics and learners.
+
+
+You can also filter or sort posts:
+
+* Filter by status: *Unread*, *Following*, *Unanswered* or *Not responded*.
+* Filter by type: *Discussion* or *Question*.
```

Comment:
I would switch this and the above, because the Type is the first thing on the left in the screenshot (so order Type, Status, Activity)

<img width="390" height="173" alt="image" src="https://github.com/user-attachments/assets/13dccab4-119a-4f69-b9a7-de463b3a2858" />

Reply:
Done